src/audiences/edit/components/estimate.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { getEstimate } from '../data';
+import { Sparklines, SparklinesLine } from 'react-sparklines';
+
+const { __ } = wp.i18n;
+
+const Estimate = props => {
+	const {
+		audience,
+		title,
+		sparkline = true,
+	} = props;
+
+	const [ loading, setLoading ] = useState( true );
+
+	const [ estimate, setEstimate ] = useState( {
+		count: 0,
+		total: 0,
+		histogram: [],
+	} );
+
+	const fetchEstimate = () => {
+		( async () => {
+			const estimateResponse = await getEstimate( audience );
+			setEstimate( estimateResponse );
+			setLoading( false );
+		} )();
```

Comment:
This immediately-invoked function is a bit confusing; can we just use regular promises instead?

Reply:
I'd not really used hooks before - this was following the docs as far as I could tell. What do you mean by using regular promises?